sycl/test-e2e/LLVMIntrinsicLowering/bitreverse.cpp

```diff
@@ -0,0 +1,213 @@
+// Test that llvm.bitreverse is lowered correctly by llvm-spirv
```

Comment:
The comment is misleading. I don't see how this test verifies that. There is no guarantee that DPC++ compiler uses llvm.bitreverse intrinsic for the source code version below.

Reply:
Expected lowered functions are checked in the SPV dump.  If DPC++ compiler does not generate LLVM IR with the expected llvm.bitreverse.* intrinsics, the SPV dump will not have the expected lowerings and the test will fail.